google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/it/BuiltinMetricsIT.java

```diff
@@ -58,71 +89,122 @@ public class BuiltinMetricsIT {
   private static final Logger logger = Logger.getLogger(BuiltinMetricsIT.class.getName());
 
   @Rule public Timeout globalTimeout = Timeout.seconds(900);
-  private static Table table;
-  private static BigtableTableAdminClient tableAdminClient;
-  private static MetricServiceClient metricClient;
+
+  private Table tableCustomOtel;
+  private Table tableDefault;
+  private BigtableDataClient clientCustomOtel;
+  private BigtableDataClient clientDefault;
+  private BigtableTableAdminClient tableAdminClient;
+  private BigtableInstanceAdminClient instanceAdminClient;
+  private MetricServiceClient metricClient;
+
+  private InMemoryMetricReader metricReader;
+  private String appProfileCustomOtel;
+  private String appProfileDefault;
 
   public static String[] VIEWS = {
     "operation_latencies",
     "attempt_latencies",
     "connectivity_error_count",
-    "application_blocking_latencies"
+    "application_blocking_latencies",
   };
 
-  @BeforeClass
-  public static void setUpClass() throws IOException {
+  @Before
+  public void setup() throws IOException {
     assume()
         .withMessage("Builtin metrics integration test is not supported by emulator")
         .that(testEnvRule.env())
         .isNotInstanceOf(EmulatorEnv.class);
 
-    // Enable built in metrics
-    BigtableDataSettings.enableBuiltinMetrics();
-
     // Create a cloud monitoring client
     metricClient = MetricServiceClient.create();
 
     tableAdminClient = testEnvRule.env().getTableAdminClient();
+    instanceAdminClient = testEnvRule.env().getInstanceAdminClient();
+    appProfileCustomOtel = PrefixGenerator.newPrefix("test1");
+    appProfileDefault = PrefixGenerator.newPrefix("test2");
+    instanceAdminClient.createAppProfile(
+        CreateAppProfileRequest.of(testEnvRule.env().getInstanceId(), appProfileCustomOtel)
+            .setRoutingPolicy(
+                AppProfile.SingleClusterRoutingPolicy.of(testEnvRule.env().getPrimaryClusterId()))
+            .setIsolationPolicy(AppProfile.StandardIsolationPolicy.of(AppProfile.Priority.LOW)));
+    instanceAdminClient.createAppProfile(
+        CreateAppProfileRequest.of(testEnvRule.env().getInstanceId(), appProfileDefault)
+            .setRoutingPolicy(
+                AppProfile.SingleClusterRoutingPolicy.of(testEnvRule.env().getPrimaryClusterId()))
+            .setIsolationPolicy(AppProfile.StandardIsolationPolicy.of(AppProfile.Priority.LOW)));
+
+    metricReader = InMemoryMetricReader.create();
```

Comment:
Explain why we're using InMemoryMetricReader and we're testing custom otel and default otel.